fingerprint2.js

```diff
@@ -52,7 +52,18 @@
       return target;
     },
     get: function(done){
-      var keys = [];
+      var that = this;
+      var keys = {
+          data: [],
+          addPreprocessedComponent: function(pair){
+              var componentKey = pair.key;
```

Comment:
IIUC this alias isn't necessary, we can simply use `pair.key` everywhere, is that correct?